content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +597,299 @@ As part of the Peruvian localization, besides creating credit notes from an exis
 
 By default the Debit Note is set in the document type.
+
+Electronic delivery guide 2.0
+-----------------------------
+
+The *Guía de Remisión Electrónica* (GRE) is an electronic document generated by the shipper to
+support the transportation or transfer of goods from one place to another, such as a warehouse or
+establishment. In Odoo, there are several steps needed in configurations before you can successfully
+use this feature.
+
+The use of the guía de remisión electrónica electronic document is mandatory and required by |SUNAT|
+for taxpayers who need to transfer their products, with the exception of those under the *Single
+Simplified Regime* (régimen único simplificado or RUS).
+
+Delivery guide types
+~~~~~~~~~~~~~~~~~~~~
+
+Sender
+******
+
+The *Sender* delivery guide type is issued when a sale is made, a service is rendered (including
+processing), goods are assigned for use, or goods are transferred between premises of the same
+company and others.
+
+This delivery guide is issued by the owner of the goods (i.e., the sender) at the beginning of the
+shipment. The sender delivery guide is supported in Odoo.
+
+.. seealso::
+   `SUNAT guía de remisión <https://www.gob.pe/7899-guia-de-remision>`_
+
+Carrier
+*******
+
+The *Carrier* delivery guide type justifies the transportation service performed by the driver (or
+carrier).
+
+This delivery guide is issued by the carrier and must be issued to each shipper when the shipment
+goes through public transport.
+
+.. important::
+   The carrier delivery guide is **not** supported in Odoo.
+
+.. seealso::
+   `SUNAT guía de remisión transportista
+   <https://tefacturo.pe/blog/sunat/guia-de-remision-electronica/guia-de-remision-transportista/>`_
+
+Transportation types
+~~~~~~~~~~~~~~~~~~~~
+
+Private
+*******
+
+The *Private* transportation type option is used when the owner carries out the transfer of goods
+using their own vehicles. In this case, a sender's delivery guide must be issued.
+
+Public
+******
+
+The *Public* transportation type option is used when an external carrier moves the goods. In
+this case, two delivery guides must be issued: the sender's delivery guide and the carrier's
+delivery guide.
+
+Changes in version 2.0
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following changes have been made in version 2.0 of the |GRE| delivery guide.
+
+Direct submission to SUNAT
+**************************
+
+The creation of the |GRE| delivery guide in Odoo **must** be sent directly to the |SUNAT|,
+regardless of the electronic document provider: IAP, Digiflow or |SUNAT|.
+
+New required information
+************************
+
+Version 2.0 requires additional information on the general configuration, vehicles, contacts, and
+products. In the general configurations, it is now necessary to add new credentials that are
+acquired in the |SUNAT| portal.
+
+Flow
+****
+
+.. image:: peru/peru-flow-remision.png
+   :align: center
+   :alt: Flow for Guia de Remision emission.
```

Comment:
Would it be possible just to have the flow (with little more details if necessary) without the picture? My concern is that I'm not sure where the picture comes from if you made it or found it, and if anything in the process changes in the next version it might be challenging to keep up with such screenshots.

Reply:
Hello @chiaraprattico @masi-odoo after some review with the Peru localization expert we concluded the "Flow" section could be deprecated, since the screenshot and the steps are redundant to the documentation itself. The steps are already explained in other sections. Thanks!